### PR TITLE
[CARBONDATA-667] Set carbon properties location proper

### PR DESCRIPTION
**Pull request by ravipesala**

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/692/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/695/

File: integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala [hunk 1]
```diff
@@ -39,14 +42,15 @@ object CarbonThriftServer {
...
-      System.setProperty("carbon.properties.filepath",
-        sparkHome + '/' + "conf" + '/' + "carbon.properties")
+      val file = new File(sparkHome + '/' + "conf" + '/' + "carbon.properties")
+      if (file.exists()) {
+        builder.config("carbon.properties.filepath", file.getCanonicalPath)
+        System.setProperty("carbon.properties.filepath", file.getCanonicalPath)
+      }
     }
-    CarbonProperties.getInstance().addProperty("carbon.storelocation", args.head)
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.STORE_LOCATION, args.head)
```
Comment (chenliang613) [inline integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala]: CarbonCommonConstants.STORE_LOCATION also get value from carbon.properties.carbon.storelocation, why need do this change ?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/712/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/716/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/720/

File: integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala (hunk 1 above)
Comment (ravipesala) [inline integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala]: it is just a constant. replaced hardcoded string to constant

**Comment by gvramana** [comment]
LGTM

**Pull request closed without merging**